docs/api/resource.rst

```diff
@@ -140,6 +140,16 @@ Filtering, sorting and paginating can all be combined together.
 * ``/resource?_sort=-last_modified&_limit=100``
 
 
+HTTP Status Codes
+-----------------
+
+* ``200 OK``: The request have been processed
+* ``304 Not Modified``: Collection items did not change since ``If-Unmodified-Since`` header value
+* ``400 Bad Request``: The request body is invalid
+* ``412 Precondition Failed``: Collection items changed since provided ``If-Unmodified-Since`` header value
+* ``503 Service Unavailable``: A backend is unavailable
```

Comment:
This could be something else: http://pretty-rfc.herokuapp.com/RFC2616#status.503